src/test/java/org/broadinstitute/consent/http/service/DacServiceTest.java

```diff
@@ -286,41 +290,43 @@ public void testRemoveDacChairFailure() {
         doNothing().when(voteService).deleteOpenDacVotesForUser(any(), any());
         initService();
 
-        service.removeDacMember(role, chair, dac);
-        verify(dacDAO, times(0)).removeDacMember(anyInt());
-        verify(voteService, times(0)).deleteOpenDacVotesForUser(any(), any());
+        assertThrows(BadRequestException.class, () -> {
+            service.removeDacMember(role, chair, dac);
+            verify(dacDAO, times(0)).removeDacMember(anyInt());
+            verify(voteService, times(0)).deleteOpenDacVotesForUser(any(), any());
```

Comment:
minor: perhaps we shouldn't focus on logic of tests, but maybe move these to outside the assert throws? so that these are actually checked.

Reply:
Agreed that I don't want to focus on the test logic in this PR.